test/built-ins/DataView/prototype/getFloat32/toindex-byteoffset.js

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-dataview.prototype.getfloat32
+description: >
+  ToIndex conversions on byteOffset
+info: |
+  24.2.4.5 DataView.prototype.getFloat32 ( byteOffset [ , littleEndian ] )
+
+  1. Let v be the this value.
+  2. If littleEndian is not present, let littleEndian be false.
+  3. Return ? GetViewValue(v, byteOffset, littleEndian, "Float32").
+
+  24.2.1.1 GetViewValue ( view, requestIndex, isLittleEndian, type )
+
+  ...
+  4. Let getIndex be ? ToIndex(requestIndex).
+  ...
+features: [DataView.prototype.setUint8]
+---*/
+
+var buffer = new ArrayBuffer(12);
+var sample = new DataView(buffer, 0);
+
+sample.setUint8(0, 75);
+sample.setUint8(1, 75);
+sample.setUint8(2, 76);
+sample.setUint8(3, 76);
+sample.setUint8(4, 75);
+sample.setUint8(5, 75);
+sample.setUint8(6, 76);
+sample.setUint8(7, 76);
+
+var obj1 = {
+  valueOf: function() {
+    return 3;
```

Comment:
I like your idea to use distinct return values for `toString` and `valueOf`--is there any reason that you did not make this change for the other tests?